simpeg/electromagnetics/natural_source/receivers.py

```diff
@@ -725,69 +498,293 @@ def evalDeriv(self, src, mesh, f, du_dm_v=None, v=None, adjoint=False):
-        \begin{bmatrix} E_x^{(x)} & E_x^{(y)} \\ E_y^{(x)} & E_y^{(y)} \end{bmatrix}_b^{-1}
+        \begin{bmatrix} T_{zx} & T_{zy} \end{bmatrix} =
+        \begin{bmatrix} H_x^{(x)} & H_y^{(x)} \\ H_x^{(y)} & H_y^{(y)} \end{bmatrix}^{-1} \,
+        \begin{bmatrix} H_z^{(x)} \\ H_z^{(y)} \end{bmatrix}
 
     where superscripts :math:`(x)` and :math:`(y)` denote signals corresponding to
     incident planewaves whose electric fields are polarized along the x and y-directions
-    respectively. Note that in simpeg, natural source EM data are defined according to
+    respectively. Note that in ``simpeg``, natural source EM data are defined according to
     standard xyz coordinates; i.e. (x,y,z) is (Easting, Northing, Z +ve up).
 
+    The receiver class can also be used to simulate a diverse set of Tipper-like data types
+    when horizontal magnetic fields are measured at a remote base station. These are defined
+    according to:
+
+    .. math::
+        \begin{bmatrix} T_{xx} & T_{yx} & T_{zx} \\ T_{xy} & T_{yy} & T_{zy} \end{bmatrix} =
+        \begin{bmatrix} H_x^{(x)} & H_y^{(x)} \\ H_x^{(y)} & H_y^{(y)} \end{bmatrix}_b^{-1} \,
+        \begin{bmatrix} H_x^{(x)} & H_y^{(x)} & H_z^{(x)} \\ H_x^{(y)} & H_y^{(y)} & H_z^{(y)} \end{bmatrix}_r
+
+    where subscript :math:`b` denotes the base station location and subscript
+    :math:`r` denotes the mobile receiver location.
+
     Parameters
     ----------
-    locations : (n_loc, n_dim) numpy.ndarray
-        Locations where the electric and magnetic fields are measured.
-        If electric and magnetic fields are measured in different locations, please
-        use `locations_e` and `locations_h` when instantiating.
-    orientation : {'xx', 'xy', 'yx', 'yy', 'zx', 'zy'}
-        Admittance receiver orientation. Specifies the admittance tensor element
-        :math:`Y_{ij}` corresponding to the data. The data type is specified by
-        the `component` input argument.
-    component : {'real', 'imag'}
-        Admittance data type. For the admittance element :math:`Y_{ij}` specified by the
-        `orientation` input argument, the receiver can be set to compute the following:
-        - 'real': Real component of the admittance (A/V)
-        - 'imag': Imaginary component of the admittance (A/V)
-    locations_e : (n_loc, n_dim) numpy.ndarray
-        Locations where the horizontal electric fields are measured.
-        Must be same size as `locations_h`.
-    locations_h : (n_loc, n_dim) numpy.ndarray
-        Locations where the magnetic fields are measured.
-        Must be same size as `locations_e`.
-    return_complex : bool, optional
-        If ``True`` the complex admittance is evaluation, and the `component` input
-        argument is ignored. Do not use for inversion!
+    locations_r : (n_loc, n_dim) array_like
+        Locations where the roving magnetic fields are measured.
+    locations_b : (n_loc, n_dim) array_like, optional
```

Comment:
I would still really like to have this be locations_bs. This is because the 'b' in this case does not refer to magnetic flux density. I recall similar conversations about avoiding the use of 'e', 'j', 'h' or 'b' when the abbreviation doesn't refer to a field.